src/fklearn/causal/cate_learning/meta_learners.py

```diff
@@ -264,3 +260,195 @@ def p(new_df: pd.DataFrame) -> pd.DataFrame:
+
+    References:
+    [1] https://matheusfacure.github.io/python-causality-handbook/21-Meta-Learners.html
+    [2] https://causalml.readthedocs.io/en/latest/methodology.html
+
+    Parameters
+    ----------
+
+    df : pd.DataFrame
+        A Pandas' DataFrame with features and target columns.
+        The model will be trained to predict the target column
+        from the features.
+
+    treatment_col: str
+        The name of the column in `df` which contains the names of
+        the treatments or control to which each data sample was subjected.
+
+    control_name: str
+        The name of the control group.
+
+    prediction_column : str
+        The name of the column with the predictions from the provided learner.
+
+    learner: LearnerMutableParametersFnType
+        A fklearn classification learner function.
+
+    treatment_learner: LearnerFnType
+        An optional fklearn classification learner function.
+
+    learner_transformers: list
+        A list of fklearn transformer functions to be applied after the learner and before estimating the CATE.
+        This parameter may be useful, for example, to estimate the CATE with calibrated classifiers.
+    """
+
+    control_learner = copy.deepcopy(learner)
+
+    if treatment_learner is None:
+        treatment_learner = copy.deepcopy(learner)
+
+    control_features = _get_learner_features(control_learner)
+    treatment_features = _get_learner_features(treatment_learner)
+
+    # pipeline
+    if learner_transformers is not None:
+        learner_transformers = copy.deepcopy(learner_transformers)
+        control_learner_pipe = build_pipeline(
+            *[control_learner(features=control_features)] + learner_transformers
+        )
+
+        treatment_learner_pipe = build_pipeline(
+            *[treatment_learner(features=treatment_features)] + learner_transformers
+        )
+    else:
+        control_learner_pipe = control_learner(features=control_features)
+        treatment_learner_pipe = treatment_learner(features=treatment_features)
+
+    # learners
+    unique_treatments = _get_unique_treatments(df, treatment_col, control_name)
+
+    learners = _get_learners(
+        df=df,
+        unique_treatments=unique_treatments,
+        control_learner=control_learner_pipe,
+        control_name=control_name,
+        treatment_learner=treatment_learner_pipe,
+        treatment_col=treatment_col,
+    )
+
+    def p(new_df: pd.DataFrame) -> pd.DataFrame:
+        return _simulate_t_learner_treatment_effect(
+            new_df,
+            learners,
+            unique_treatments,
+            control_name,
+            prediction_column,
+        )
+
+    p.__doc__ = learner_pred_fn_docstring("causal_t_classification_learner", shap=True)
+    partial_log = {
+        "causal_features": {
+            "control": control_features,
+            "treatment": treatment_features,
+        },
+    }
+
+    log = {"causal_t_classification_learner": partial_log}
```

Comment:
As as suggestion, we could get the logs of each learner and something like the following:

```
 log = {
        "causal_t_classification_learner": {
            **learners_logs,
            "causal_features": control_features,
            "treatment_features": treatment_features,
        }
    }
```